### PR TITLE
Admin Site: add column in table view

### PR DESCRIPTION
**Pull request by aramis-eb**
Add column admin site in a table to open the Event Dashboard on Admin

### CODE REVIEW

File: search_events_app/services/db/db_response_processor.py [hunk 1]
```diff
@@ -42,3 +43,8 @@ def get_organizer(item):
     if item[5]:
         return item[5]
     return None
+
+
+def get_admin_url(item):
+    url = "https://www.eventbrite.com/myevent?slow_load=1&eid=" + f'{item[0]}'
```
Comment (mmonzo-eb) [inline search_events_app/services/db/db_response_processor.py:49]: Don't create a new variable, just return 'https...'
Use single quotes for string and format everything in one single string like this:
f'https://www.eventbrite.com/myevent?slow_load=1&eid={item[0]}'

File: tests/services/db/test_db_response_processor.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ def test_process_events(self):
 			'format_': 'Meeting or Networking Event',
 			'organizer': 'Adam Kurtz Org',
 			'country': 'United States',
+            'admin_url': 'https://www.eventbrite.com/myevent?slow_load=1&eid=99894936444',
```
Comment (mmonzo-eb) [inline tests/services/db/test_db_response_processor.py:48]: I see this wrongly indented. I don't know if it's because of github's format

Comment (aramis-eb) [reply tests/services/db/test_db_response_processor.py:48]: Yes it's github's format

File: search_events_app/services/db/db_response_processor.py (hunk 1 above)
Comment (aramis-eb) [reply search_events_app/services/db/db_response_processor.py:49]: Resolved!

File: tests/services/db/test_db_response_processor.py (hunk 2 above)
Comment (aramis-eb) [reply tests/services/db/test_db_response_processor.py]: Resolved

**Pull request merged**